azerothcore/azerothcore-wotlk
# fix(DB/spell_dbc): The Green Tower equip spell trigger

**Description:**
Issue was reported saying the green tower(30-39) shield spell proc was not going off at all.
Investigated the spell inside spell editor and saw inconsistent values on the trigger chances.
I changed the values by making comparisons to skullflame shield and the issue appears to be fixed.



## Changes Proposed:
-  Database spell overrides for the TheGreenTower aura ability and its related trigger.

## Issues Addressed:

- Closes #5649

## Tests Performed:
Issue built without errors, I personally did the repro steps below and got 2 procs back to back after some time.

## How to Test the Changes:


1. Spawn a Shaman in durotar
2. Make him level 60
3. Give yourself item 1204
4. Equip item 1204
5. Run up to a boar
6. punch it in the face
7. let it auto attack you 
8. observe you will get equip proc eventually 

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
